doc/source/getting_started/comparison/comparison_with_sql.rst

```diff
@@ -233,6 +233,13 @@ default, :meth:`~pandas.DataFrame.join` will join the DataFrames on their indice
 parameters allowing you to specify the type of join to perform (``LEFT``, ``RIGHT``, ``INNER``,
 ``FULL``) or the columns to join on (column names or indices).
 
+.. warning::
+
+    If both key columns contain rows where the key is ``NA`` or ``NaN``, those
```

Comment:
instead of explicitly listing the null values, just say null values.

Reply:
See updated PR - is this better?